Makefile

```diff
@@ -269,6 +272,12 @@ dummy := $(shell (export ROCKSDB_ROOT="$(CURDIR)"; \
 # this file is generated by the previous line to set build flags and sources
 include make_config.mk
 
+ROCKSDB_PLUGIN_MKS = $(foreach plugin, $(ROCKSDB_PLUGINS), plugin/$(plugin)/*.mk)
+include $(ROCKSDB_PLUGIN_MKS)
```

Comment:
I am a little confused as to how one specifies ROCKSDB_PLUGINS and gets the plugin/* directory in the corresponding source tree.  More documentation would be helpful.

Reply:
It's created by the user.